src/lib/ngsi/ContextElement.cpp

```diff
@@ -112,6 +112,25 @@ std::string ContextElement::render(ConnectionInfo* ciP, RequestType requestType,
 
 /* ****************************************************************************
 *
+* ContextElement::getAttribute
+*/
+ContextAttribute* ContextElement::getAttribute(std::string attrName)
```

Comment:
Would probably have been better to implement a 'lookup' method for ContextAttributeVector, but, never mind.

NTC